src/jukebox/components/playermpd/__init__.py

```diff
@@ -447,6 +447,16 @@ def play_card(self, folder: str, recursive: bool = False):
             self.second_swipe_action()
         else:
             logger.debug('Calling first swipe action')
+
+            # sync audiofolder
+            _files_synced = plugs.call_ignore_errors('sync_shared', 'ctrl', 'sync_folder', args=[folder])
```

Comment:
I haven't found a better place for this, as the synchronisation is needed before the folder is actually played. 
If i would call this before playermpd is invoked (in e.g. the rfid_reader), i would have fiddle with the decoded [card_action](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/e4e718ec8c85d257397b5723042b6d31a3d25a98/src/jukebox/components/rfid/reader/__init__.py#L219) to see if play_card is invoked and which args is the folder name. And i personally see this as a nogo.

If the plugin isn't enabled (as in no 'modules' entry in jukebox.yaml) the error is ignored and playback is not harmed, but it would still be logged.
I think here comes my question 1 from the first post in place, on how to make the calls best.